src/app/components/footer.tsx

```diff
@@ -0,0 +1,93 @@
+"use client";
+
+import { motion } from "framer-motion";
+import dynamic from "next/dynamic";
+import { useEffect, useState } from "react";
+
+const FaSoundcloud = dynamic(() =>
+  import("react-icons/fa").then((mod) => mod.FaSoundcloud)
+);
+const FaInstagram = dynamic(() =>
+  import("react-icons/fa").then((mod) => mod.FaInstagram)
+);
+const FaArrowUp = dynamic(() =>
+  import("react-icons/fa").then((mod) => mod.FaArrowUp)
+);
+
+const BackToTopButton = () => {
+  const [isVisible, setIsVisible] = useState(false);
+
+  useEffect(() => {
+    const toggleVisibility = () => {
+      if (window.pageYOffset > 300) {
+        setIsVisible(true);
+      } else {
+        setIsVisible(false);
+      }
+    };
+
+    window.addEventListener("scroll", toggleVisibility);
+
+    return () => window.removeEventListener("scroll", toggleVisibility);
+  }, []);
+
+  const scrollToTop = () => {
+    window.scrollTo({
+      top: 0,
+      behavior: "smooth",
+    });
+  };
+
+  return (
+    <>
+      {isVisible && (
+        <button onClick={scrollToTop} className="fixed bottom-0">
+          <FaArrowUp />
+        </button>
+      )}
+    </>
+  );
+};
+
+const Footer = () => {
+  return (
+    <motion.footer
+      id="contact"
+      className="pb-10 md:pb-16"
```

Comment:
I think I would like to have a little padding on both top and bottom, to look like something like that:

<img width="424" alt="image" src="https://github.com/user-attachments/assets/9f07745d-e989-4243-a974-5eac7248a6be" />

Maybe this 👇 What do you think?

```suggestion
      className="py-10 md:py-16"
```